pom.xml

```diff
@@ -155,6 +157,12 @@
 				<artifactId>spring-cloud-starter-common-security-config-web</artifactId>
 				<version>${spring-cloud-common-security-config.version}</version>
 			</dependency>
+			<!-- looks like starter doesn't override version pulled by skipper? do this temporarily for now -->
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-common-security-config-web</artifactId>
+				<version>${spring-cloud-common-security-config.version}</version>
```

Comment:
(I'm aware that this is not SB 2.1 yet), Just be aware that this implementation may clash with the SB 2.1's ManagementWebSecurityAutoConfiguration!  For similar use case Artem suggested @AutoConfigureBefore(ManagementWebSecurityAutoConfiguration)